src/processor/intents/creeps/_die.js

```diff
@@ -12,6 +12,26 @@ module.exports = function(object, roomObjects, bulk, stats, dropRate) {
     bulk.remove(object._id);
     delete roomObjects[object._id];
 
+    let tombstone = {
+        type: 'tombstone',
+        room: object.room,
+        x: object.x,
+        y: object.y,
```

Comment:
`x`, `y`, and `room` are the same as in the tombstone, no need to store them.

Reply:
Those are the tombstone `x`, `y`, and `room`